src/api/app/models/event/create_package.rb

```diff
@@ -19,7 +19,7 @@ def metric_tags
     end
 
     def metric_fields
-      { value: 1 }
+      { project: payload['project'], package: payload['package'], count: Package.count }
```

Comment:
why `payload['project']`? didn't we say we don't want this info there? :thinking:

Reply:
Yes, but we change our mind :grinning:

Reply:
we? ... you! explain me why! :wink:

Reply:
We will be able to extract the info from influxdb to another tool as you said when we started to discuss this :grin: :laughing:

Reply:
Ah! If I was right... then I'll approve it.... :laughing: